serialized_data_converter.py

```diff
@@ -349,6 +349,7 @@ def convert(self, edit):
                 self.output = yaml.yaml_dumps(
                     self.plist,
                     default_flow_style=default_flow_style,
+                    indent=self.settings.get("yaml_indent"),
```

Comment:
I know the settings file provides a default, but please provide the default in the source as well like is done everywhere else.